roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -91,8 +91,8 @@ cifmw_ceph_rgw_config:
 cifmw_cephadm_cephfs_name: "cephfs"
 cifmw_cephadm_ceph_spec_fqdn: "{{ ceph_spec_fqdn | default(false) | bool }}"
 cifmw_ceph_dashboard_spec_path: /tmp/ceph_dashboard.yml
-cifmw_cephadm_certificate: ""
-cifmw_cephadm_key: ""
```

Comment:
Please update to set these back to the empty string:
```yaml
cifmw_cephadm_certificate: ""
cifmw_cephadm_key: ""
```
I will take care of new defaults for the above in https://github.com/openstack-k8s-operators/ci-framework/pull/1865 when it's ready.

Reply:
@fultonj yes i updated them